src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -6751,7 +6752,7 @@ namespace VariableSpeedCoils {
 
 	//     NOTICE
 
-	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
+	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
```

Comment:
@Myoldmopar every time I open this in xcode and save it, it changes the the &copy; to some unknown character. Not sure how to stop it from doing that.